src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -552,7 +552,6 @@ class MoreInfoLight extends LitElement {
 
       .content > * {
         width: 100%;
-        max-height: 84px;
```

Comment:
I don't think this is the right solution, I remember it was there for a reason... Will check... We could at least remove `max-height` from `ha-attributes`